### ISSUE TITLE
 broken link

### ISSUE THREAD

**Issue by Yakovlev-Melarn** [phalcon/cphalcon#12215]
A link in the upper right corner of the debug component refers to the 404 page.

![screenshot at sep 12 22-03-56](https://cloud.githubusercontent.com/assets/1256298/18448743/e733f540-7934-11e6-8360-d04fbe637e82.png)

**Comment by stamster** [comment]
Correct. Missing http:// link prefix.

```
     <div class="version">Phalcon Framework 
            <a target="_new" href="//docs.phalconphp.com/en/3.0.1/">3.0.1</a>
        </div>
```

**Comment by stamster** [comment]
+1

Also for v.2.0.13: https://docs.phalconphp.com/en/2.0.13/

**Comment by tembem** [comment]
I guess this https://github.com/phalcon/cphalcon/blob/master/phalcon/debug.zep#L295

**Comment by sergeyklay** [comment]
Fixed in the `3.0.x` branch.
